lib/package/plugins/policyMetaData.json

```diff
@@ -48,10 +52,11 @@
   "SpikeArrest" :{"indications":["spikearrest","spike","sa","^spikearrest$"]},
   "StatisticsCollector" :{"indications":["stats","statcoll"], "profiles":["apigee"] },
   "TraceCapture" :{"indications":["tracecapture","tc","^tracecapture$","^tc$"], "profiles":["apigeex"] },
+  "ValidateSAMLAssertion" :{"indications":["validatesaml","saml","sa"]},
   "VerifyAPIKey" :{"indications":["verifyapikey","apikey","va","verify","^verifyapikey$"]},
   "VerifyJWS" :{"indications":["verifyjws","jws","vjws","^verifyjws$"]},
   "VerifyJWT" :{"indications":["verifyjwt","jwt","vjwt","^verifyjwt$"]},
-  "XMLThreatProtection" :{"indications":["xmltp","tp"]},
+  "XMLThreatProtection" :{"indications":["xmltp","xtp","tp","^xmltp$"]},
   "XMLToJSON" :{"indications":["xmltojson","x2j","^xmltojson$"]},
```

Comment:
Can we replace
```"XMLToJSON" :{"indications":["xmltojson","x2j","^xmltojson$"]},``` with
```"XMLToJSON" :{"indications":["xmltojson","x2j","xtoj","^xmltojson$"]},``` 
(added xtoj to be consistent with JSONToXML)

Similarly can we replace
```  "JSONToXML" :{"indications":["jsontoxml","j2x","jtox"]},``` with 
```  "JSONToXML" :{"indications":["jsontoxml","j2x","jtox","^jsontoxml$"]},```
(added ^jsontoxml$ to be consistent with XMLToJSON

Reply:
yes, good catch.